test/tests/nek_errors/deformation/box-test1.i

```diff
@@ -50,6 +45,7 @@
   [heat]
     type = HeatConduction
     variable = temp
+    use_displaced_mesh = true
```

Comment:
Ooh, if it's going to be this much repetition then I will backpedal and say let's go with the `[GlobalParams]` - I thought we'd just need `use_displaced_mesh = true` in the `[Mesh]` block. 

Sorry for the whiplash!

Reply:
Yeah, it gets a bit tedious. This will take some time to test (turning it off for each of these objects, seeing where behaviour diverges from expectations). So far we've just been using GlobalParams at the top as a redundant safety net, but it does create issues when used with the Tensor Mechanics module and its dependencies.

No worries, I'll undo the change.